src/rtc/h5_prep.py

```diff
@@ -603,80 +708,117 @@ def get_metadata_dict(product_id: str,
-         ' bounding polygon']
-
-    metadata_dict['identification/burstID'] = \
-        ['burst_id', str(burst_in.burst_id),
-         'Burst identification (burst ID)']
-
-    beam_id = burst_in.swath_name.upper()
-    metadata_dict['identification/beamID'] = \
-        ['beam_id', beam_id, 'Beam identification (Beam ID)']
-
-    metadata_dict['identification/zeroDopplerStartTime'] = \
-        ['zero_doppler_start_time',
-         burst_in.sensing_start.strftime(DATE_TIME_METADATA_FORMAT),
-         'Azimuth start time of the product in the format YYYY-MM-DDThh:mm:ss.sZ'] # 1.6.3
-    metadata_dict['identification/zeroDopplerEndTime'] = \
-        ['zero_doppler_end_time',
-         burst_in.sensing_stop.strftime(DATE_TIME_METADATA_FORMAT),
-         'Azimuth stop time of the product in the format YYYY-MM-DDThh:mm:ss.sZ']  # 1.6.3
-
-    metadata_dict['metadata/sourceData/zeroDopplerStartTime'] = \
-        [None,
-         burst_in.sensing_start.strftime(DATE_TIME_METADATA_FORMAT),
-         'Azimuth start time of the product in the format YYYY-MM-DDThh:mm:ss.sZ'] # 1.6.3
-    metadata_dict['metadata/sourceData/zeroDopplerEndTime'] = \
-        [None,
-         burst_in.sensing_stop.strftime(DATE_TIME_METADATA_FORMAT),
-         'Azimuth stop time of the product in the format YYYY-MM-DDThh:mm:ss.sZ']  # 1.6.3
-    metadata_dict['metadata/sourceData/numberOfAzimuthLines'] = \
-        ['source_data_number_of_azimuth_lines',
-         burst_in.length,
-         'Number of azimuth lines within the source data product']
-
-    metadata_dict['metadata/sourceData/numberOfRangeSamples'] = \
-        ['source_data_number_of_range_samples',
-         burst_in.width,
-         'Number of slant range samples for each azimuth line within the source data']
-    metadata_dict['metadata/sourceData/slantRangeStart'] = \
-        ['source_data_slant_range_start',
-         burst_in.starting_range,
-         'Source data slant range start distance']
-
-    # Add RFI metadata into `metadata_dict`
-    rfi_metadata_dict = get_rfi_metadata_dict(burst_in,
-                                              'metadata/QA/rfiInformation')
-    metadata_dict.update(rfi_metadata_dict)
-
-    return metadata_dict
+    if not is_mosaic:
+
+        # Metadata only for for burst product
+        # Calculate bounding box
+        xmin_geogrid = cfg_in.geogrids[str(burst_in.burst_id)].start_x
+        ymax_geogrid = cfg_in.geogrids[str(burst_in.burst_id)].start_y
+        spacing_x = cfg_in.geogrids[str(burst_in.burst_id)].spacing_x
+        spacing_y = cfg_in.geogrids[str(burst_in.burst_id)].spacing_y
+        width_geogrid = cfg_in.geogrids[str(burst_in.burst_id)].width
+        length_geogrid = cfg_in.geogrids[str(burst_in.burst_id)].length
+        xy_bounding_box = [
+            xmin_geogrid,
+            ymax_geogrid + length_geogrid * spacing_y,
+            xmin_geogrid + width_geogrid * spacing_x,
+            ymax_geogrid
+        ]
```

Comment:
Is the bounding box based on the SRS (i.e. EPSG) of the geogrid? if yes, I wonder if there is any need to provide EPSG for the geogrid.